go.sum

```diff
@@ -1,17 +1,22 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
the vendor folder didn't change so why do we have a diff here?

Reply:
I did not change this, maybe `make` command changed this. 
Should I revert this change manually?